### DISCUSSION TITLE
decativate environment variables

### DISCUSSION THREAD

**Discussion by gandhi404** [mediagis/nominatim-docker#240] [Q&A]
Hello,
is it possible to exchange the environment variable PBF_URL for a local path with the * .pbf and to deactivate the replication completely?

best regards

**Comment by leonardehrenfried** [comment]
At the moment this is not possible but that doesn't mean that it cannot be contributed. 

I could imagine a way where anything that starts with `http` is being downloaded and anything that starts with `/` is treated as a local file inside the docker container. You would have to mount the file into the container for this to work.

If you want to deactivate the replication you can write a garbage URL in there and just never trigger the command. (Nominatim doesn't automatically replicate - only when you tell it to.)

**Reply by gandhi404** [reply]
what do I have to change or set so that he does not attempt a replication?

**Reply by leonardehrenfried** [reply]
REPLICATION_URL=http://example.com and then do nothing.

**Reply by gandhi404** [reply]
thx!
